dojo/tools/sarif/parser.py

```diff
@@ -185,18 +196,19 @@ def get_item(result, rules, artifacts, run_date):
     # we add a special 'None' case if there is no CWE
     cwes = [0]
     if rule is not None:
```

Comment:
You have 2 times `if rule is not None:`, here and in line 179. Can you put the code under one `if`?

Reply:
modified.